lldb/test/API/python_api/sbframe_extensions/TestSBFrameExtensions.py

```diff
@@ -0,0 +1,548 @@
+"""
+Test SBFrameExtensions API.
+"""
+
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class TestSBFrameExtensions(TestBase):
+    def setUp(self):
+        TestBase.setUp(self)
+        self.source = "main.c"
+
+    def test_properties_pc_addr_fp_sp(self):
+        """Test SBFrame extension properties: pc, addr, fp, sp"""
+        self.build()
+        self.setTearDownCleanup()
+        exe = self.getBuildArtifact("a.out")
+
+        target, process, thread, bkpt = lldbutil.run_to_source_breakpoint(
+            self, "Set breakpoint here", lldb.SBFileSpec(self.source)
+        )
+
+        frame = thread.GetFrameAtIndex(0)
+        self.assertTrue(frame.IsValid(), "Frame should be valid")
+
+        # Test pc property
```

Comment:
Comments should be English prose, so starting with a capital letter (which you did) and ending with a period (which is missing). Applies to the whole PR.